Improve RST and MD files

felixfontein opened the pull request:
Subset of #1287 that doesn't add linting.

#1272 should be merged first and this PR then updated.

hiddeco commented:
@felixfontein any chance you would be up to addressing the mixed usage of Sops / SOPS / sops throughout the document in this PR? In my opinion, it should be either "SOPS" (the name) or `sops` (the binary) depending on the context.

felixfontein commented:
@hiddeco good idea, I added a commit which normalize the SOPS usages. I converted quite a few `sops` to SOPS as well. Please take a look whether you think that's OK.

hiddeco commented on README.rst:
```diff
@@ -331,79 +333,79 @@ We assume you have an instance (or more) of Vault running and you have privilege
 ~~~~~~~~~~~~~~~~~~~~~~~~
 
-When creating new files, ``sops`` uses the PGP, KMS and GCP KMS defined in the
+When creating new files, SOPS uses the PGP, KMS and GCP KMS defined in the
```
As we later make mention of flags, maintaining the command reference is better here I think.

There are a couple of other places where this applies as well.

felixfontein replied:
I hope I manged to fix the right ones you meant.

hiddeco reviewed (approved):
![](https://media.giphy.com/media/26n08WnCiKbJ4l2Ffv/giphy.gif)

felixfontein commented:
@hiddeco thanks for reviewing and merging!
